lua/nvim-tree/lib.lua

```diff
@@ -81,6 +81,16 @@ function M.get_last_group_node(node)
   return next
 end
 
+function M.get_all_nodes_in_group(node)
```

Comment:
We could simplify a bit `open_all_nodes_in_group` howeve this function will likely see use in the future.

Reply:
Would a function to "apply function to all nodes in group" be more useful instead maybe?

Reply:
> Would a function to "apply function to all nodes in group" be more useful instead maybe?

See `NodeIterator`

You could use that here.